### ISSUE TITLE
Wrong version on docker hub

### ISSUE THREAD

**Issue by Wykks** [RocketChat/hubot-rocketchat#137]
Currently the dockerfile from docker hub (`rocketchat/hubot-rocketchat`) install the lastest version of hubot-rocketchat
But this version doesn't work with the current version of rocket chat (as you mentionned in the README file), it's stuck to this message : 
`Subscribing to Room: __my_messages__`

The Dockerfile from this repo is fine, but the README tell us to use the version from the docker hub `rocketchat/hubot-rocketchat`

Proposition : 
https://hub.docker.com/r/singli/hubot-rocketchat/~/dockerfile/
`npm install hubot-rocketchat`
Should be modified to something like this : 
`npm install hubot-rocketchat@0.1.4`

**Comment by geekgonecrazy** [comment]
@Wykks 
The dockerfile being used to build is: https://hub.docker.com/r/rocketchat/hubot-rocketchat/~/dockerfile/

We recently made a change from using npm inside of our container to compiling the current code at the time of a tag.

We were running into issues of the version inside the container not being the version we tagged.  So this was the most reliable solution.

Check out the tags here:
https://hub.docker.com/r/rocketchat/hubot-rocketchat/tags/

Any tag here will contain the relevant code for that version.

So if you want to run version v0.1.4 just use this image: `rocketchat/hubot-rocketchat:v0.1.4`
